django_mongodb/query.py

```diff
@@ -302,3 +305,111 @@ def register_nodes():
     Join.as_mql = join
     NothingNode.as_mql = NothingNode.as_sql
     WhereNode.as_mql = where_node
+
+
+class MongoQuerySet(QuerySet):
+
+    def raw_mql(self, raw_query, params=(), translations=None, using=None):
+        return MongoRawQuerySet(
+            raw_query,
+            model=self.model,
+            params=params,
+            translations=translations,
+            using=using,
+        )
+
+
+class MongoRawQuery(RawQuery):
+
+    def __init__(self, sql, using, model, params=()):
+        super().__init__(sql, using, params)
+        self.model = model
+
+    def __iter__(self):
+        self.cursor = self._execute_query()
+        if not connections[self.using].features.can_use_chunked_reads:
+            result = list(self.cursor)
+        else:
+            result = self.cursor
+        return iter(result)
+
+    def _execute_query(self):
+        connection = connections[self.using]
+        collection = connection.get_collection(self.model._meta.db_table)
+        return collection.aggregate(self.sql)
+
+    def get_columns(self):
+        if self.cursor is None:
+            self._execute_query()
+        return [f.column for f in self.model._meta.fields]
+
+    def __str__(self):
+        return "%s" % self.sql
+
+
+class MongoRawQuerySet(RawQuerySet):
+    def __init__(
+        self,
+        raw_query,
+        model=None,
+        query=None,
+        params=(),
+        translations=None,
```

Comment:
Not sure if it will fix `test_translations`, but you fail to pass `translations` to super()`.

Reply:
Good catch! Didn't fix the test but may as well pass to super() 569719b527bcc1f6c6493ff7c517847d7ddbf416